1_September/js_assignment.html

```diff
@@ -0,0 +1,262 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>js Assignment</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
+  </head>
+  <body>
+    <h2>Javascript Assignment</h2>
+
+
+
+    <script>
+        //1) Create a function that takes two numbers as arguments and returns their sum.
+        // function addition(num1,num2){
+        //   return num1+num2    
+        // }
+        // const test1=console.log(addition(3, 2))
+        // const test2=console.log(addition(-3, -6))
```

Comment:
create seperate script folder and create script file for each function.

Reply:
created new PR